### PR TITLE
TVB-2700: Fixed noise configuration in case of Stochastic Integrators + Tests

### PR DESCRIPTION
**Pull request by VinczeRobert**
Did some additional cleanup on unit-tests around this area

### CODE REVIEW

**Comment by VinczeRobert** [comment]
> These look good to me, I did some tests and everything seems to work correctly.
> Please fix related test and we can merge.

Ok, but this test is failing in master as well...

**Comment by popaula937** [comment]
> > These look good to me, I did some tests and everything seems to work correctly.
> > Please fix related test and we can merge.
> 
> Ok, but this test is failing in master as well...

Yes, but we have to fix all tests eventually, and would probably be easier if we do it incrementally,  by fixing the ones that are related to our tasks.

**Comment by VinczeRobert** [comment]
I fixed the test and I know that something's not good as there are a lot more failures on Jenkins, I will try to see why...

**Comment by VinczeRobert** [comment]
Please don't merge this yet, because I see there are more tests failed on Jenkins, I think I have added clean_database() where I should have not, I will check again...

File: framework_tvb/tvb/tests/framework/interfaces/web/controllers/noise_configuration_controller_test.py [hunk 1]
```diff
@@ -34,67 +34,35 @@
...
+        self.noise_controller = NoiseConfigurationController()
+        SimulatorController().index()
+        self.simulator = cherrypy.session[common.KEY_SIMULATOR_CONFIG]
+        self.connectivity = connectivity_factory()
+        self.simulator.connectivity = self.connectivity.gid
+        self.simulator.integrator = HeunStochastic()
 
     def transactional_teardown_method(self):
         """ Cleans the testing environment """
+        self.clean_database()
```
Comment (liadomide) [inline framework_tvb/tvb/tests/framework/interfaces/web/controllers/noise_configuration_controller_test.py:62]: Why this cleanup? considering that the test in transactional ?

**Comment by VinczeRobert** [comment]
I deleted transactional_setup_fixtures and I might have found the problem, or at least one of them, but I am not quite sure, I have to investigate some more and I am curious how many tests fail now on Jenkins.

**Pull request merged**